quick-start/src/main/java/com/marklogic/quickstart/service/MappingManagerService.java

```diff
@@ -66,10 +69,6 @@ public MappingModel createMapping(String projectDir, MappingModel newMapping) th
         scaffolding.createMappingDir(newMapping.getName());
         Mapping mapping = mappingManager.createMappingFromJSON(newMapping.toJson());
         mappingManager.saveMapping(mapping);
```

Comment:
Should this piece of code be removed ? After a mapping is created, we do have to install user modules right ? Also deleteMapping() still unwatches the mapping dir when called